src/__test__/components.test.js

```diff
@@ -0,0 +1,40 @@
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
- [ ] Since this is a capstone project it would be appreciated to add more than 7 tests.